src/Miunie.Discord/Configuration/IBotConfiguration.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Miunie.Discord.Configuration
+{
+    public interface IBotConfiguration
+    {
+        string GetBotToken();
+    }
+}
```

Comment:
Whops, missed a newline here.

You can actually set it to add a newline at the end of a file in VS.